apps/docs/app/ui/layout/sectionPopover/sectionPopover.css

```diff
@@ -1,25 +1,25 @@
+.hd-section-popover {
+    width: calc(100% - (var(--hd-space-2) * 2));
```

Comment:
this should match the popover's containerPadding, but it doesn't. RACPopover has a prop for this.